mne/viz/evoked.py

```diff
@@ -60,6 +61,40 @@ def _butterfly_on_button_press(event, params):
     params['need_draw'] = False
 
 
+def _butterfly_onselect(xmin, xmax, ch_types, evoked):
+    """Function for drawing topomaps from the selected area."""
+    import matplotlib.pyplot as plt
+    times = evoked.times
+    xmin *= 0.001
+    xmin = times.flat[np.abs(times - xmin).argmin()]
+    minidx = np.where(times == xmin)[0][0]
+    xmax *= 0.001
+    xmax = times.flat[np.abs(times - xmax).argmin()]
+    maxidx = np.where(times == xmax)[0][0]
+
+    fig, axarr = plt.subplots(1, len(ch_types))
+    axarr = np.atleast_1d(axarr)
```

Comment:
Instead of these two lines, you can do `...subplots(..., squeeze=False)[0]` and it should give you a list of the first-row axes.